Telemetry.ps1

```diff
@@ -139,7 +139,8 @@ function Invoke-SendTelemetryEvent
         Invoke-* methods share a common base code.  Leaving this as-is to make this file
         easier to share out with other PowerShell projects.
 #>
-    [CmdletBinding(SupportsShouldProcess)]
```

Comment:
This change is causing the tests to fail right now, because we still have the check at line 157 that calls `$PSCmdlet.ShouldProcess()`:

```
Invoke-SendTelemetryEvent' calls ShouldProcess/ShouldContinue but does not have the ShouldProcess attribute.

Stack trace
at line: 157 in /home/vsts/work/1/s/Telemetry.ps1
 ShouldProcess
 Severity: Warning
```

Reply:
Removed `ShouldProcessBlock`